Dockerfile.cr

```diff
@@ -40,7 +40,7 @@ COPY --from=build /configs/ /configs/
 COPY --from=istio-1_11_4 /usr/local/bin/istioctl /bin/istioctl-1.11.4
 COPY --from=istio-1_12_3 /usr/local/bin/istioctl /bin/istioctl-1.12.3
 # For multiple istioctl binaries, provide their paths separated with a colon (:) like in the Linux PATH variable.
```

Comment:
- Please remove this comment if changed to `;` as separator.
- Is this ENV value only used by that `parsePaths` function? If it is, then it doesn't have to change, right? If it's for compatible with windows path, then will this change break linux path recognition?

Reply:
I will change the comment. It does not break linux, it was just for staying consistent with linux env var guidelines, but it will only be used with parsePath function. In the CLi scenario, this env var will be set by Kyma CLI, but since Windows Default path is something like `C:\<...>`, we need to change to a symbol which is also not allowed in Windows paths